### PR TITLE
[OPB-582] Update installation documentation for OP Worker 1.1

### PR DESCRIPTION
**Pull request by binarylogic**
This updates the Observability Pipelines Worker installation steps for the new 1.1 release.

### CODE REVIEW

**Comment by binarylogic** [comment]
Just a note. I shied away from doing too much cleanup on the gitlab file and opened https://datadoghq.atlassian.net/browse/OPB-637. I can do this as part of the next release.

File: content/en/observability_pipelines/installation.md [hunk 1]
```diff
@@ -18,10 +18,11 @@ Install the Observability Pipelines Worker with the [Advanced Package Tool][1] (
 
 ## Prerequisites
 
-Before installing, make sure you have:
+Before installing, make sure you:
 
-1. A valid [Datadog API key][5].
-2. An Observability Pipelines Configuration ID.
+1. Are using one of the supported Linux Archs: x86_64 or AMD64
```
Comment (buraizu) [inline content/en/observability_pipelines/installation.md:23]:
```suggestion
1. Are using one of the supported Linux architectures: x86_64 or AMD64
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for the PR, just left a small suggestion

**Comment by binarylogic** [comment]
@buraizu @maycmlee could I get a review on this so I can merge with the 1.1 OPW release? Thanks.

**Review by maycmlee — APPROVED** [review]

**Pull request merged**